### DISCUSSION TITLE
sort helper

### DISCUSSION THREAD

**Discussion by louiskhchan** [mockoon/mockoon#1257] [Ideas]
Hi,

Is it possible to add a custom helper for sorting an array? 

I'm aware of the new filtering, sorting and pagination feature on the GET route. However, it uses specific URL parameter patterns, which may not match the existing usage in my environment. Currently, I can read the query parameters and use the array helpers such as `filter` and `slice` to simulate such functions, but a `sort` helper is missing. It would be nice if you have plans to implement it.

Thank you :)

**Comment by 255kb** [comment]
Hi, there is no such plan currently, but feel free to open an issue, it would indeed be useful! And as always contributions are welcomed :)

**Reply by louiskhchan** [reply]
Thank you. I opened an issue at:
https://github.com/mockoon/mockoon/issues/1270
